### ISSUE TITLE
Add a -B, --bind-to argument to create/update sandbox

### ISSUE THREAD

**Issue by dshafik** [akamai/cli-sandbox#36]
It would be great if you could specify the `host` setting when creating and updating a sandbox (and wherever else it's appropriate).

Alternatively, defaulting to binding to `0.0.0.0` would be far better in our case.

**Comment by krzyk** [comment]
@dshafik Did you mean starting sandbox connector (the host specified in the client json file)? I don't see options that would allow binding in case of update/create sandbox.

**Comment by dshafik** [comment]
@krzyk after almost two years since I wrote this and 18 months since I left Akamai, I am not 100% certain but, yes, I believe it was instead of having to update the config file by hand.

**Comment by krzyk** [comment]
@dshafik Yeah I know, just wanted to confirm what I was guessing, thanks for quick reply :)
